package.json

```diff
@@ -37,12 +38,13 @@
     "super-request": "^1.2.0"
   },
   "scripts": {
-    "prepush": "npm run lint && npm run test",
+    "prepushe": "npm run lint && npm run test",
     "start:prod": "concurrently \"npm run server:prod\" \"npm run client\"",
     "start": "concurrently \"npm run server\" \"npm run client\"",
-    "server": "cross-env NODE_ENV=development node index.js",
+    "server": "cross-env NODE_ENV=development supervisor index.js",
     "server:prod": "cross-env NODE_ENV=production node index.js",
-    "client": "node start-client.js",
+    "cliente": "node start-client.js",
+    "client": "cd frontend && npm start",
```

Comment:
I hope this works fine because previously the client process was started as a subprocess.

Reply:
It should work, unless I don't understand what you mean..